content/22.vaccines.md

```diff
@@ -2,8 +2,20 @@
 
 ### Abstract
 
+The release of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genome sequence on January 10th, 2020, mobilised global vaccine development on a scale never before seen in response to the coronavirus disease 2019 (COVID-19) pandemic. 
+Pre-existing vaccine platforms and novel vaccine technologies were deployed by manufacturers early in the pandemic. 
+Regardless of the platform used, almost all the vaccine candidates have targeted the spike protein of SARS-CoV-2. 
+A historically slow process, vaccine development accelerated to the point that less than a year later, some vaccine candidates had reported interim phase III clinical trial data and received emergency use authorization (EUA) in the United States and approval by the European Union. 
+In this review, the strategies used to develop the leading vaccine candidates are discussed and their safety and efficacy are appraised.
+Currently, publically available safety and efficacy data generated from the phase III clinical trials supports the distribution of spike targeting adenoviral vector vaccines and mRNA vaccines.
+Lastly, the initial state of global vaccine distribution is also reviewed.
+
 ### Importance
 
+SARS-CoV-2 has infected over 125 million people and cost the lives of 2.8 million people globally. 
+The development, production, and distribution of prophylactic vaccines is imperative to save lives, prevent illness, and reduce the economic and social burdens caused by the COVID-19 pandemic. 
```

Comment:
We may be able to be even stronger here.  My sense is that non-pharmaceutical interventions were the primarily option for preventing illness and saving lives during the first "phase" of the pandemic.  The arrival of safe, effective vaccines has moved us to the next "phase", in which we can work to end the pandemic.

Reply:
In relation to @ajlee21 comment: “Please correct me if I'm wrong, but my understanding is that not all the vaccines used the genome sequence, but this sequence allowed for the development of novel vaccines that target SARS-CoV2. If that's the case I might re-word this to focus on the effort towards rapid vaccine development in response to the pandemic without mentioning the sequencing...”


You’re not wrong, but the overwhelming majority did as they targeted the spike protein and therefore needed to know the sequence, e.g. mRNA, viral vector, and protein vaccines all required the sequence. Unless I’m wrong, I think the only vaccines in distribution that did not require information on spike was the inactivated vaccines: both sinopharm vaccines, covaxin (Indian candidate) and a few others that have not reached phase III. A good overview by the NYT here:  https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html

I think this line still stands as all of the major vaccines in distribution with phase III supported data required the sequence, but I am happy to amend if we think this is incorrect as it stands. 

Thanks for reviewing @agitter @ajlee21

Reply:
```suggestion
The development, production, and distribution of prophylactic vaccines is imperative to saving lives, preventing illness, and reducing the economic and social burdens caused by the COVID-19 pandemic.
Assuming effective deployment, these vaccines offer an opportunity to move into a new phase of the pandemic where the susceptibility of worldwide populations is significantly reduced. 
```
Taking a stab at addressing @agitter's comment

Reply:
Thanks a million @rando2 , I couldn’t have put it better